### PR TITLE
added composer config support for twig v3

### PR DESCRIPTION
**Pull request by Duneyr**
Hi Adrian,

many thanks for your work! I'd like to use twig v3 also and your tests are doing great with twig 3.0.3.

Best regards
Bodo

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adriansuter/twig-cache-busting",
+    "name": "duneyr/twig-cache-busting",
```
Comment (adriansuter) [inline composer.json]: The package name should be left as is.

**Review by adriansuter — CHANGES_REQUESTED** [review]
Thanks.

File: composer.json (hunk 1 above)
Comment (Duneyr) [reply composer.json]: Sorry!!!!! Wrong branch!!!! I'll change it!!!

**Review by adriansuter — APPROVED** [review]

**Comment by Duneyr** [comment]
So should be the initial change again. Sorry again, I do not do PR a lot on github....

**Comment by adriansuter** [comment]
Thank you. I will immediately create a new release.

**Pull request merged**